pkg/integration/catch_all_test.go

```diff
@@ -0,0 +1,34 @@
+package integration
+
+import (
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/cmsgov/easi-app/pkg/handlers"
+)
+
+func (s *IntegrationTestSuite) TestCatchAllRoute() {
+	s.Run("get /", func() {
+		req, err := http.NewRequest("GET", "/notapath", nil)
+		s.NoError(err)
+		rr := httptest.NewRecorder()
```

Comment:
The integration tests don't need the `httptest` recorder. The suite setup runs the server so you can run a request directly against that like in the system intake tests: https://github.com/CMSgov/easi-app/blob/master/pkg/integration/system_intakes_test.go#L38. No need to run the handler in here either